contrib/container/Dockerfile.alpine

```
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1.17
+
+ARG ALPINE_VERSION
+ARG ALPINE_ARCH
+
+FROM scratch AS root
+ARG ALPINE_VERSION
+ARG ALPINE_ARCH
+# alpine does not publish loong64 images on docker.io
+ADD --unpack=true https://dl-cdn.alpinelinux.org/alpine/latest-stable/releases/${ALPINE_ARCH}/alpine-minirootfs-${ALPINE_VERSION}-${ALPINE_ARCH}.tar.gz /
```

Hadolint (2.14.0)

[error] invalid flag: --unpack

(DL1000)